susemanager-utils/susemanager-sls/salt/images/kiwi-image-build.sls

```diff
@@ -35,20 +45,24 @@ mgr_buildimage_prepare_activation_key_in_source:
 
 mgr_buildimage_kiwi_prepare:
   cmd.run:
-    - name: "kiwi --nocolor --force-new-root --prepare {{ source_dir }} --root {{ chroot_dir }} {{ kiwi_params }}"
+    - name: "{{ kiwi }} --nocolor --force-new-root --prepare {{ source_dir }} --root {{ chroot_dir }} {{ kiwi_params() }}"
     - require:
       - module: mgr_buildimage_prepare_source
       - file: mgr_buildimage_prepare_activation_key_in_source
 
 mgr_buildimage_kiwi_create:
   cmd.run:
-    - name: "kiwi --nocolor --yes --create {{ chroot_dir }} --dest {{ dest_dir }} {{ kiwi_params }}"
+    - name: "{{ kiwi }} --nocolor --yes --create {{ chroot_dir }} --dest {{ dest_dir }} {{ kiwi_params() }}"
     - require:
       - cmd: mgr_buildimage_kiwi_prepare
 
 mgr_buildimage_kiwi_bundle:
   cmd.run:
-    - name: "kiwi --nocolor --yes --bundle-build {{ dest_dir }} --bundle-id {{ bundle_id }} --destdir {{ bundle_dir }} {{ kiwi_params }}"
+{%- if have_bundle_build %}
+    - name: "{{ kiwi }} --nocolor --yes --bundle-build {{ dest_dir }} --bundle-id {{ bundle_id }} --destdir {{ bundle_dir }}"
+{%- else %}
+    - name: "mkdir -p '{{ bundle_dir }}' && cd '{{ dest_dir }}' && tar czf '{{ bundle_dir }}'`basename *.packages .packages`-{{ bundle_id }}.tgz --no-recursion `find . -maxdepth 1 -type f` && cd '{{ bundle_dir }}' && sha256sum *.tgz > `echo *.tgz`.sha256"
```

Comment:
Can you please insert a comment on why do we need this bundle?

Reply:
We need the bundle because image collect is based upon assumption we get only one file from build host which is tgz with both system and boot images.
But this oneliner is quite massive. What do you think about moving the `{%- if have_bundle_build %}` condition around whole `mgr_buildimage_kiwi_bundle` state and prepare different one with multiple `cmd.run` states in it?